flight-website/app/styles/app.css

```diff
@@ -76,6 +76,24 @@ main.ds-main {
   font-bold;
 }
 
+.ds-beta-banner {
+  display: block;
```

Comment:
Question: div tags have display: block by default. Is it necessary to include this style?

Reply:
True. Fixed now.